packages/core/src/extension/extension-action-creator.ts

```diff
@@ -34,4 +37,38 @@ export class ExtensionActionCreator {
                     });
             });
     }
+
+    renderExtension(
+        container: string,
+        region: ExtensionRegions,
+    ): ThunkAction<ExtensionAction, InternalCheckoutSelectors> {
+        return (store) =>
+            Observable.create((observer: Observer<ExtensionAction>) => {
+                const state = store.getState();
+                const { id: cartId } = state.cart.getCartOrThrow();
+                const extensions = state.extensions.getExtensions();
+                const extension = extensions?.filter((e) => e.region === region)[0];
```

Comment:
I know right now we only support one extension, but this will need to be changed if we have two exensions both for different regions. 
Maybe we need to add a method called `extensions.getExtensionByRegion` which takes a region as an argument and returns the extension for that region:
`state.extensions.getExtensionByRegion(region: ExtensionRegion): Extension`

Reply:
Thanks. I have added `state.extensions.getExtensionByRegion(region: ExtensionRegion): Extension` and renamed `ExtensionRegions` to `ExtensionRegion`.